### ISSUE TITLE
Task - Geographic Area Lookup: sorting option

### ISSUE THREAD

**Issue by AntWeb-org** [SpeciesFileGroup/taxonworks#3253]
### Feature or enhancement

Would like to have a sorting option under Details for display columns, e.g., sort by latitude, longitude, country, county, state.

@bpescador, @cgrinter, @rich-keller

### Location

Task - Geographic Area Lookup

**Comment by mjy** [comment]
Is the use case for this to see what coverage we have in the GeographicArea table?  Since this task isn't related to specimen data per-se (see other comments) the use case for this is unclear.  There is minimal information about Geographic Areas in /show (including children, etc.).

**Comment by AntWeb-org** [comment]
Yes, it would be a quick and easy way to see what is available and where. If I want to check for all San Jose GAs assuming that name is widely used, sorting would be a quick way to visually group all San Jose GAs from the United States or South America.

**Comment by cgrinter** [comment]
@AntWeb-org - not sure how I got pulled into this, but please stop mentioning me - I didn't request to follow this issue and have no idea what it is.

**Comment by debpaul** [comment]
> @AntWeb-org - not sure how I got pulled into this, but please stop mentioning me - I didn't request to follow this issue and have no idea what it is.

@cgrinter you can customize this for yourself from the GitHub repository. Look at the top of this page, where it says "Unwatch" and use the dropdown to choose your settings (which apply to the entire repository). You're in this thread because you are mentioned in the top of the ticket. Hope this helps.
![image](https://user-images.githubusercontent.com/2990155/208493639-1a771885-3bad-443f-bd4b-23c6b245bd6f.png)

**Comment by AntWeb-org** [comment]
> > @AntWeb-org - not sure how I got pulled into this, but please stop mentioning me - I didn't request to follow this issue and have no idea what it is.
> 
> @cgrinter you can customize this for yourself from the GitHub repository. Look at the top of this page, where it says "Unwatch" and use the dropdown to choose your settings (which apply to the entire repository). You're in this thread because you are mentioned in the top of the ticket. Hope this helps. (Or we can edit the top of the ticket, where Brian, as AntWeb can remove your name). ![image](https://user-images.githubusercontent.com/2990155/208493639-1a771885-3bad-443f-bd4b-23c6b245bd6f.png)

Thank you @debpaul! Apparently there are two C. Grinters and we've been tagging the wrong one. Apologies and Happy Holidays!!

**Comment by AntWeb-org** [comment]
not intended use
